cfme/tests/cloud/test_tenant_quota.py

```diff
@@ -107,30 +112,35 @@ def test_tenant_quota_enforce_via_lifecycle_cloud(request, appliance, provider,
 # second arg is a list of lists, with each one a test is to be generated
 # sequence is important here
 # indirect is the list where we define which fixtures are to be passed values indirectly.
-@pytest.mark.skip('Catalog items are converted to collections. Refactoring is required')
 @pytest.mark.parametrize(
-    ['set_roottenant_quota', 'custom_prov_data', 'extra_msg'],
+    ['set_roottenant_quota', 'custom_prov_data', 'extra_msg', 'context'],
     [
-        [('cpu', 2), {}, ''],
-        [('storage', 0.001), {}, ''],
-        [('memory', 2), {}, ''],
-        [('vm', 1), {'catalog': {'num_vms': '4'}}, '###']
+        [('cpu', 2), {}, '', ViaUI],
+        [('cpu', 2), {}, '', ViaSSUI],
+        [('storage', 0.001), {}, '', ViaUI],
+        [('storage', 0.001), {}, '', ViaSSUI],
+        [('memory', 2), {}, '', ViaUI],
+        [('memory', 2), {}, '', ViaSSUI],
+        [('vm', '1'), {'catalog': {'num_vms': '4'}}, '###', ViaUI],
+        [('vm', '1'), {'catalog': {'num_vms': '4'}}, '###', ViaSSUI]
     ],
-    indirect=['set_roottenant_quota'],
-    ids=['max_cpu', 'max_storage', 'max_memory', 'max_vms']
+    indirect=['set_roottenant_quota', 'custom_prov_data'],
+    ids=['max_cpu-ViaUI', 'max_cpu-ViaSSUI', 'max_storage-ViaUI', 'max_storage-ViaSSUI',
+         'max_memory-ViaUI', 'max_memory-ViaSSUI', 'max_vms-ViaUI', 'max_vms-ViaSSUI']
 )
 def test_tenant_quota_enforce_via_service_cloud(request, appliance, provider, setup_provider,
-                                          set_roottenant_quota, extra_msg, custom_prov_data,
-                                          prov_data, template_name, catalog_item):
-    """Test Tenant Quota in UI"""
-    catalog_item.provisioning_data.update(custom_prov_data)
-    catalog_item.provisioning_data['catalog']['vm_name'] = catalog_item.vm_name
-    catalog_item.provisioning_data.update({'properties': {
-        'instance_type': partial_match('m1.large')}})
-    catalog_item.create()
-    service_catalogs = ServiceCatalogs(appliance, catalog_item.catalog,
-                                       catalog_item.name)
-    service_catalogs.order()
+                                                set_roottenant_quota, extra_msg, context,
+                                                custom_prov_data, template_name, catalog_item):
+    """Test Tenant Quota in UI and SSUI"""
```

Comment:
**REQUIRED:**
Ideally, this test should look like this
```python
        with appliance.context.use(context):
            service_catalogs = ServiceCatalogs(appliance, catalog_item.catalog,
                                           catalog_item.name)
            service_catalogs.order()
```
If adding to shopping cart is a required setup, it should be handled inside the order, also, why does the args change if the context is different?